conda-requirements.txt

```diff
@@ -2,7 +2,7 @@
 # conda create -n <name> --file conda-requirements.txt
 
 # Mandatory dependencies
-biggus>=0.12.0
+biggus>=0.13.0
```

Comment:
It does do what I want, I've checked it locally, it just isn't going to get tested on Travis CI because it picks the latest version which is v0.13.0 (see https://travis-ci.org/SciTools/iris/jobs/135538831#L330).

Regardless this line does need to be changed in minimal requirements file, I  will do this.